### ISSUE TITLE
URGENT: Add mean epsilon for each r-m bin in deagg response data

### ISSUE THREAD

**Issue by nluco-usgs** [usgs/nshmp-haz-ws#137]
Analogous to the mean epsilon returned for the largest r-m bin, e.g. ...

                {
                  "name": "ε₀",
                  "value": 1.36,
                  "units": "σ"
                },

... add the mean epsilon for each of the other r-m bins, e.g. ...

      "data": [
        {
          "component": "Total",
          "data": [
            {
              "r": 10.0,
              "m": 8.3,
              "εdata": [
                {
                  "εbin": 8,
                  "value": 3.9742642
                },
                {
                  "εbin": 9,
                  "value": 1.3679938
                },
                {
                  "εbin": 10,
                  "value": 3.8023821E-4
                }
              ]
            },

Consider also adding the mean r and mean m for each of the other r-m bins (in addition to the mode).

@pmpowers-usgs

**Comment by pmpowers-usgs** [comment]
The data bins will be updated to, for example:
```
  "data": [
    {
      "component": "Total",
      "data": [
        {
          "r": 10.0,
          "m": 8.3,
          "rBar": 9.56,
          "mBar": 8.12,
          "εBar": 1.34,
          "εdata": ...
```
These additional fields shouldn't have any impact on UHT rendering.

**Comment by pmpowers-usgs** [comment]
Resolved in usgs/nshmp-haz#331
